exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function here
-
+function createLongGreeting (a, b){
+    return "Hello, my name is " + a + " and I'm " + b + " years old"
```

Comment:
This is good! However, this may also be a nice place to try out _string interpolation_ (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).